src/dataviews/histogram-dataview-model.js

```diff
@@ -37,11 +35,14 @@ module.exports = DataviewModelBase.extend({
           params.push('offset=' + offset);
         }
       }
-      if (_.isNumber(start)) {
-        params.push('start=' + start);
+
+      // Start - End
+      var limits = this._originalData.getCurrentStartEnd();
+      if (limits !== null && _.isNumber(limits.start)) {
```

Comment:
Would be nice to unify the two `limits !== null` conditionals as we have in `histogram-data-model`